test/src/tests-forwarders.js

```diff
@@ -991,13 +994,9 @@ describe('forwarders', function() {
 
         done();
     });
+    });
 
     it('sends events to forwarder v1 endpoint when mParticle.config.isDevelopmentMode = config.isDebug = false', function(done) {
-        fetchMock.post(urls.forwarding, {
```

Comment:
Why is fetchMock being removed?

Reply:
it was moved up to the `beforeEach` block.